VEHICLE/SetVehicleHeadlightsColour.md

```diff
@@ -0,0 +1,32 @@
+---
+ns: VEHICLE
+aliases: ["0xE41033B25D003A07"]
+---
+## _SET_VEHICLE_HEADLIGHTS_COLOUR
+
+```c
+// 0xE41033B25D003A07
+void _SET_VEHICLE_HEADLIGHTS_COLOUR(Vehicle vehicle, int color);
+```
+
+```
+color is the paint index for the vehicle.
+Paint index goes from 0 to 12.
+
+Be aware that it only works on xenon lights.
+```
+
+**Result**:
+>! ![](https://i.imgur.com/yV3cpG9.png)
+
+## Examples
```

Comment:
Unsure, but don't examples usually go _after_ parameters/return values in these files? Let's stick to convention.